lib/Mojolicious/Static.pm

```diff
@@ -146,7 +148,9 @@ sub warmup {
 
   # DATA sections
   my $index = $self->{index} = {};
-  for my $class (reverse @{$self->classes}) { $index->{$_} = $class for keys %{data_section $class} }
+  for my $class (reverse @{$self->classes}) {
+    $index->{$_} = $class for keys %{data_section $class};
+  }
```

Comment:
Same, does nothing. Why is it included?

Reply:
My perltidy runs on save. It must think that is not valid given the perltidyrc, I'll revert it.